iidm/iidm-api/src/main/java/com/powsybl/iidm/network/NetworkListener.java

```diff
@@ -36,6 +36,14 @@ default void onElementRemoved(Identifiable identifiable, String attribute, Objec
         // empty default implementation
     }
 
+    default void onElementConnected(Connectable<?> connectable) {
+        // empty default implementation
+    }
+
+    default void onElementDisconnected(Connectable<?> connectable) {
```

Comment:
I'm not sure of what is needed here ? Connectable is enough ?

Reply:
Why do we not use an attribute like fir the other notifications? Why a specific method?

Reply:
done